src/openforms/js/lang/nl.json

```diff
@@ -65,7 +65,7 @@
     "originalDefault": "Incomplete Submissions Removal Method"
   },
   "20K9To": {
-    "defaultMessage": "Service fetch configuration",
+    "defaultMessage": "Servicebevraging instellen",
```

Comment:
Isn't 'fetch' an acceptable Dutch term instead of bevraging?

Reply:
I stuck to this term as that's what we've communicated so far about this feature.